File: src/java/pe/edu/upeu/application/model/Horario.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package pe.edu.upeu.application.model;

/**
 *
 * @author Admin
 */
public class Horario {

    private String id_horario;
    private String ho_desde;
    private String ho_hasta;
    private String dia_horario;
    private String es_horario;
    private String id_detalle_horario;

    public Horario(String id_horario, String ho_desde, String ho_hasta, String dia_horario, String es_horario, String id_detalle_horario) {
        this.id_horario = id_horario;
        this.ho_desde = ho_desde;
        this.ho_hasta = ho_hasta;
        this.dia_horario = dia_horario;
        this.es_horario = es_horario;
        this.id_detalle_horario = id_detalle_horario;
    }

    public Horario() {
        this.id_horario = "";
        this.ho_desde = "";
        this.ho_hasta = "";
        this.dia_horario = "";
        this.es_horario = "";
        this.id_detalle_horario = "";
    }

    public String getId_horario() {
        return id_horario;
    }

    public void setId_horario(String id_horario) {
        this.id_horario = id_horario;
    }

    public String getHo_desde() {
        return ho_desde;
    }

    public void setHo_desde(String ho_desde) {
        this.ho_desde = ho_desde;
    }

    public String getHo_hasta() {
        return ho_hasta;
    }

    public void setHo_hasta(String ho_hasta) {
        this.ho_hasta = ho_hasta;
    }

    public String getDia_horario() {
        return dia_horario;
    }

    public void setDia_horario(String dia_horario) {
        this.dia_horario = dia_horario;
    }

    public String getEs_horario() {
        return es_horario;
    }

    public void setEs_horario(String es_horario) {
        this.es_horario = es_horario;
    }

    public String getId_detalle_horario() {
        return id_detalle_horario;
    }

    public void setId_detalle_horario(String id_detalle_horario) {
        this.id_detalle_horario = id_detalle_horario;
    }

}

File: src/main/webapp/js/businessLogic/Horario/horario.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

var l = [];
l[0] = "lun";
l[1] = "mar";
l[2] = "mie";
l[3] = "jue";
l[4] = "vie";
l[5] = "sab";
l[6] = "dom";
var lb = [];
lb[0] = "Lunes";
lb[1] = "Martes";
lb[2] = "Miércoles";
lb[3] = "Jueves";
lb[4] = "Viernes";
lb[5] = "Sabado";
lb[6] = "Domingo";
function listHorario(id_dgp) {
    var url = '../../horario?iddgp=' + id_dgp;
    var data = 'opc=Listar2';
    $.post(url, data, function (objJson) {
        var listas = objJson.listar;
        if (listas.length > 0) {
            $(".tipoh").empty();
            $(".tipoh").append("Oficina");
            $(".conTablas").empty();
            for (var j = 0; j < l.length; j++) {
                var f = 0;
                var t = '';
                var d = 0;
                for (var i = 0; i < listas.length; i++) {
                    if (listas[i].dia_horario.toString() === l[j].toString()) {
                        d = d + 1;
                        t += '<tr>';
                        t += '<td>Turno ' + d + '</td>';
                        t += '<td>' + listas[i].ho_desde + '</td>';
                        t += '<td>' + listas[i].ho_hasta + '</td>';
                        t += '</tr>';
                        f = 1;
                    }
                }
                if (f > 0) {
                    var g = createTableHorario(j);
                    $(".conTablas").append(g);
                    $(".data" + l[j] + "").empty();
                    $(".data" + l[j] + "").append(t);
                }
            }
          //  $(".dataHor").dataTable();
        } else {
            $(".conTablas").empty();
            $(".conTablas").append('<div class="alert alert-danger">No sujeto a fiscalización</div>');
        }
    });
}
function createTableHorario(a) {
    var t = '<div class="col-md-12">';
    t += '<table class="table table-condensed table-bordered dataHor">';
    t += '<thead>';
    t += '<tr>';
    t += '<th colspan="3" style="background:#01579b ;color:white;"><center>' + lb[a] + '</center></th>';
    t += '</tr>';
    t += '</thead>';
    t += '<tbody class="data' + l[a] + '">';
    t += '</tbody>';
    t += '</table>';
    t += '</div>';
    return t;
}
